### PR TITLE
Remove unused image and add size check

### PR DESCRIPTION
**Pull request by futuretea**
After disable local-storage, servicelb, traefik, theirs image can be removed

iso file will just 2G

**Related issue**
rancher/harvester#508

### CODE REVIEW

**Comment by gitlawr** [comment]
Can we add a check to ensure the built image does not exceed the size limit?

**Comment by futuretea** [comment]
> Can we add a check to ensure the built image does not exceed the size limit?

Good idea, I will add it.

**Comment by futuretea** [comment]
> Can we add a check to ensure the built image does not exceed the size limit?

Updated

File: scripts/build [hunk 1]
```diff
@@ -85,8 +89,28 @@ go mod vendor
...
+find "${artifacts_dir}"  \( -name '*.iso' -or -name 'k3os-vmlinuz-*' -or -name 'k3os-initrd-*' \) -type f -print0 |
+  while IFS= read -r -d '' file; do
+      mv "${file}" "${file/k3os/harvester}"
+  done
+
+# check artifacts size
+max_size=2147483647
+find "${artifacts_dir}"  -type f -print0 |
+  while IFS= read -r -d '' file; do
+      size=$(du -k "${file}" | cut -f 1)
```
Comment (gitlawr) [inline scripts/build:108]: This gets the size in KB

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**